lnpos/100_config.ino

```diff
@@ -161,18 +160,23 @@ void readFiles()
     if (lnurlATM != "null" || lnurlATM != "")
     {
       baseURLATM = getValue(lnurlATM, ',', 0);
+      // remove /api/v1.... and add /atm?lightning=
+      int apiPos = baseURLATM.indexOf("api");
```

Comment:
why is this replace required? why isn't the URL correct form the start?

Reply:
its the correct url from the start for the lnurl endpont `/fossa/api/v1/lnurl/` but for the atm page we need the public page `/`fossa/atm` page